EIPS/eip-####.md

```diff
@@ -0,0 +1,335 @@
+---
```

Comment:
```suggestion
---
eip: 7657
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.